### PR TITLE
Adding azure-functions-java-library-dapr to pom.xml for java template

### PR DESCRIPTION
**Pull request by ASHIQUEMD**

### CODE REVIEW

**Comment by Flanker32** [comment]
@ASHIQUEMD Thanks a lot for your efforts and sorry for the late response.

As maven archetype did not include files based on parameter, we implement the trigger generation by groovy script, which will run `mvn azure-functions:add` in groovy script if users select template other than HttpTrigger, here is the related codes
https://github.com/microsoft/azure-maven-archetypes/blob/develop/azure-functions-archetype/src/main/resources/META-INF/archetype-post-generate.groovy

File: azure-functions-archetype/src/main/resources/archetype-resources/pom.xml [hunk 1]
```diff
@@ -32,6 +32,9 @@
 #end
 #if(${trigger.toLowerCase()}=="durablefunctions")
         <durabletask.azure.functions.version>1.0.0</durabletask.azure.functions.version>
+#end
+#if(${trigger.toLowerCase()}=="daprbindingtrigger" || ${trigger.toLowerCase()}=="daprserviceinvocationtrigger" || ${trigger.toLowerCase()}=="daprtopictrigger" || ${trigger.toLowerCase()}=="daprstateinputbinding" || ${trigger.toLowerCase()}=="daprsecret" || ${trigger.toLowerCase()}=="daprstateoutputbinding" || ${trigger.toLowerCase()}=="daprinvokeoutputbinding" || ${trigger.toLowerCase()}=="daprpublishoutputbinding" || ${trigger.toLowerCase()}=="daprbindingoutputbinding")
```
Comment (Flanker32) [inline azure-functions-archetype/src/main/resources/archetype-resources/pom.xml]: trigger refers to the name in your template, which should be `DaprTopicTrigger`, `DaprServiceInvocationTrigger`, `DaprBindingTrigger`

https://github.com/microsoft/azure-maven-plugins/pull/2356/files#diff-872e88ba5476b3792dc494696121715546141b5bb1338b913b8ef04bb1f9e9f8R458
https://github.com/microsoft/azure-maven-plugins/pull/2356/files#diff-872e88ba5476b3792dc494696121715546141b5bb1338b913b8ef04bb1f9e9f8R499
https://github.com/microsoft/azure-maven-plugins/pull/2356/files#diff-872e88ba5476b3792dc494696121715546141b5bb1338b913b8ef04bb1f9e9f8R531

Comment (ASHIQUEMD) [reply azure-functions-archetype/src/main/resources/archetype-resources/pom.xml]: Hey thank you for pointing this out, I have added the 3 template names that is part of [this project](https://github.com/microsoft/azure-maven-plugins/pull/2356/files#diff-872e88ba5476b3792dc494696121715546141b5bb1338b913b8ef04bb1f9e9f8R458).

**Review by Flanker32 — APPROVED** [review]
LGTM

**Pull request merged**